Erp connector (#418)
akeneo/pim-api-docs: content/guides/erp-connection/step3-understand-akeneo-pim.md

# Understand Akeneo PIM

To become an Akeneo PIM master, we have many different tools to help you!

## Versions and editions

First, it is crucial for you to understand that Akeneo PIM comes in all shapes and sizes, depending on its version and edition.

To know more about the different PIM versions, don't hesitate to consult our blog on our [PIM release cycle](https://www.akeneo.com/blog/akeneo-introduces-a-simpler-product-release-cycle/).

> You can also consult our [release notes](https://www.akeneo.com/release-notes/) and our ["new key features released" page](https://help.akeneo.com/pim/serenity/whats-new.html) to understand what are our latest Akeneo supported **versions** and their brand new features.

Reviewer: ```suggestion
You can also consult our [release notes](https://www.akeneo.com/release-notes/) and our ["new key features released" page](https://help.akeneo.com/pim/serenity/whats-new.html) to understand what our latest Akeneo supported **versions** and their brand new features are.
```

Then, in order to understand our different **editions** and their capabilities, you can consult our [compare editions page](https://www.akeneo.com/compare-editions/) on our website.

## PIM features and data structure

>Let's play with Akeneo PIM!

It is crucial to understand the PIM features and data model in order to create a great ERP/PIM connector.

The best way to become an expert at Akeneo PIM is to play with it!  
You can access our [Akeneo PIM Community Edition demo platform](https://demo.akeneo.com) (login/password: admin / admin).

:::warning
Please note that this Akeneo PIM CE demo instance is for exploratory purposes only and will only allow you to start getting familiar with Akeneo PIM. This instance is reset frequently and is therefore not intended to be used to build a connector.
:::

If you need an Akeneo PIM **Enterprise Edition** instance to start developing your connector, please [contact us](https://www.akeneo.com/contact/).

## Documentation

> First of all, for the documentation part, we advise you to take a look at our [Helpcenter](https://help.akeneo.com/pim/index.html) where you'll find a lot of documentation about all our features and how to use it.

Reviewer: ```suggestion
First of all, we advise you to take a look at our [Helpcenter documentation](https://help.akeneo.com/pim/index.html) about all our features and how to use them.
```

> Then, if you feel like it wasn't enough, we designed a comprehensive training program to share everything we know about Akeneo PIM. As you can see, we got you covered as well with our [training sessions](https://www.akeneo.com/training/). Don't hesitate to register!

Reviewer: ```suggestion
Then, if you feel like you need more, we designed a comprehensive training program to share everything we know about Akeneo PIM. As you can see, we also got you covered our [training sessions](https://www.akeneo.com/training/). Don't hesitate to register!
```
